src/masoniteorm/factories/Factory.py

```diff
@@ -9,44 +9,43 @@ def __init__(self, model, number=1):
         self.model = model
         self.number = number
 
-    def make(self, dictionary={}, name="default"):
+    def make(self, dictionary=None, name="default"):
+        if dictionary is None:
+            dictionary = {}
```

Comment:
What is the reason for setting this to `None` here?

Reply:
dictionary argument had a mutable default value on the method signature. using an immutable default value is recommended if you want to set dictionary argument as {} when the method is called.

Also PyCharm warns in this case:

![image](https://user-images.githubusercontent.com/53943169/100837718-2e2c9a80-3482-11eb-9c10-eb4b7eb8ca24.png)

You can see that for more details: https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
hmm I see. I'll have to test this out. I have no noticed this issue before in Python.

Wouldn't the solution instead of it defaulting to None wouldn't it be:

```python
def make(self, dictionary={}, name="default"):
        dic = {}
        dic.update(dictionary)
```

Setting it to None I feel is strange

Reply:
your solution solves the issue, too. After your test, let's apply the solution that you feel comfortable to use it.

Reply:
Is there any update? Which solution do you prefer?

Reply:
I think the example I put above where we keep everything a dictionary

Reply:
I've just tried the above solution and tests failed. This solution forces the dic variable to be a dict in all cases however it may be a list in some cases. Whereas the dictionary variable must be an empty dict when only it is None according to the logic in the method. My solution provides this requirement. What do you think about how should we proceed?

Reply:
hmm that's true. I suppose your original solution with the None type is fine

Reply:
Okay, I reverted the last 2 commits that cause tests were failed. Now, tests are passed.